CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### Unreleased
+
+- Added `Container::factory(service)`
+- Added `Container::extend(id, service)`
```

Comment:
This PR is just about `extend()`. The `factory()` was implemented here: https://github.com/gacela-project/gacela/pull/229